x-pack/plugins/ml/server/models/job_service/jobs.ts

```diff
@@ -84,10 +84,35 @@ export function jobsProvider(
     });
   }
 
-  async function deleteJobs(jobIds: string[], deleteUserAnnotations = false) {
+  async function deleteJobs(
```

Comment:
I'm seeing this error when deleting jobs (jobs I tried don't have alerting rules):

<img width="260" alt="image" src="https://github.com/elastic/kibana/assets/7405507/64514284-b91e-49c9-8320-789edef9387a">

Reply:
No longer see this with https://github.com/elastic/kibana/pull/176049/commits/b3d61c10d72e5c48d38431887a4cbc34f9ee1e5b